tools/topology/topology1/CMakeLists.txt

```diff
@@ -133,6 +133,7 @@ set(TPLGS
 	"sof-imx8-nocodec-sai\;sof-imx8-nocodec-sai"
 	"sof-imx8-wm8960\;sof-imx8-wm8960\;-DPPROC=volume"
 	"sof-imx8-wm8960\;sof-imx8-eq-iir-wm8960\;-DPPROC=eq-iir-volume"
+	"sof-imx8-wm8960\;sof-imx8-eq-fir-wm8960\;-DPPROC=eq-fir-volume"
```

Comment:
Commit message looks weird: you're looking in amixer for FIR and it's using IIR coefficients? Those don't work together normally... Everything else is good but I'm curious about that.

Reply:
True, it's just a copy paste for the example of sof-ctl tool usage taken from a previous commit. Now should be OK.